src/index.js

```diff
@@ -1 +1,27 @@
-import './styles/main.css';
\ No newline at end of file
+import './styles/main.css';
+import printElements from './utilities/printTodos.js';
+
+const todosList = [
+  {
+    content: 'something1',
+    id: new Date().toISOString(),
```

Comment:
For the `id`, it was explicitly [mentioned](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=completed%20%5Bbool%5D.-,index%3A%20%5Bnumber%5D.,-Write%20a%20function) to use a number as the type but this date method being invoked would return a string so I strongly recommend using the length of the array in order to stick to the project requirements since future features would rely on this.